packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -1128,6 +1140,48 @@ export const prepareCosmosOrchestrationAccountKit = (
             watch(E(this.facets.helper.owned()).executeEncodedTx(msgs, opts)),
           );
         },
+        /** @type {HostOf<NobleMethods['depositForBurn']>} */
+        depositForBurn(destination, amount) {
+          return asVow(() => {
+            trace('depositForBurn', { destination, amount });
+            const { helper } = this.facets;
+            const { chainAddress } = this.state;
+
+            /** @type {EthChainInfo} */
+            // @ts-expect-error cast
+            const chainInfo = chainHub.getChainInfoByChainId(
+              destination.chainId,
+            );
+            typeof chainInfo.cctpDestinationDomain === 'number' ||
+              Fail`Must be reachable by CCTP`;
+            const { cctpDestinationDomain } = chainInfo;
+
+            if (typeof cctpDestinationDomain !== 'number') {
+              throw Fail`${q(destination.chainId)} does not have "cctpDestinationDomain" set in ChainInfo`;
+            }
```

Comment:
It seems we check if `cctpDestinationDomain` is present and a number twice. Please just do it once, and prefer the second error message (q(destination.chainId)} does not have "cctpDestinationDomain" set in ChainInfo) which seems more helpful to developers.